### PR TITLE
Fix for broken offline site generation for conference events

### LINKED ISSUE #1985: Offline site generation broken for conference events
```
  File "/opt/indico/env/lib/python2.7/site-packages/indico-1.9.4-py2.7.egg/MaKaC/webinterface/pages/static.py", line 76, in getJSFiles
    return WPStaticEventBase.getJSFiles(self) + self._includeJSPackage('Timetable')
AttributeError: class WPStaticEventBase has no attribute 'getJSFiles'
```

### PR DESCRIPTION
**Pull request by mic4ael**
fixes #1985

### CODE REVIEW

File: indico/MaKaC/webinterface/pages/static.py [hunk 1]
```diff
@@ -42,6 +44,9 @@ def _getMenu(self):
             if isinstance(link, SystemLink) and link.getName() not in ContextManager.get("_menu_offline_items"):
                 link.setVisible(False)
 
+    def getJSFiles(self):
+        return (self._asset_env['base_js'].urls() + self._asset_env['modules_event_display_js'].urls())
```
Comment (ThiefMaster) [inline indico/MaKaC/webinterface/pages/static.py]: no need for the () around it

**Comment by ThiefMaster** [comment]
60 chars commit msg? :/

Why not simply "Fix offline site generation for conferences" (43 chars)

File: indico/MaKaC/webinterface/pages/static.py (hunk 1 above)
Comment (mic4ael) [inline indico/MaKaC/webinterface/pages/static.py]: Ah, yeah. Forgot to remove that.

**Comment by ThiefMaster** [comment]
Merged fbf096b

**Pull request closed without merging**